tests/unit/src/flixel/ui/FlxButtonTest.hx

```diff
@@ -14,6 +17,7 @@ class FlxButtonTest extends FlxTest
 	{
 		button = new FlxButton();
 		destroyable = button;
+		FlxG.mouse.playback(new MouseRecord(0, 0, FlxInputState.RELEASED, 0)); // put the mouse in the upper-left corner to prepare for highlight testing
```

Comment:
I just tried a new approach:

```haxe
		Lib.current.stage.removeChild(FlxG.game);
		Lib.current.stage.addChild(new FlxGame(640, 480, FlxState, 1, 60, 60, true));
```
Which works better: the FlxButtonTest and the others that broke now pass.  However, it still makes the FlxReplayTest fail with accumulated keys.  I find this strange, but we're getting closer...